add MB-System (#74)
softwareunderground/awesome-open-geoscience: README.md

- [Stress2Grid](http://pmd.gfz-potsdam.de/wsm/showshort.php?id=escidoc:2112906) – ![matlab](media/icon/matlab.png) Two concepts to calculate the mean SHmax orientation
- [QGIS](#platforms) – GIS platform see [Platforms](#platforms)
- [Verde](https://github.com/fatiando/verde) – ![Python](media/icon/python.png) processing spatial data to regular grids

> - [MB-System](https://github.com/dwcaress/MB-System) - MB-System is an open source software package for the processing and display of bathymetry and backscatter imagery data derived from multibeam, interferometry, and sidescan sonars.

Reviewer: Please follow the PR template, it would avoid many reviews. The following changes are still necessary:

- [ ] Remove "MB Systems" from description
- [ ] Remove "open source software" from description
- [ ] Fix the dash
- [ ] Add icon for "C" language
- [ ] Remove "."

- [OpendTect](https://dgbes.com/index.php/software#free) – Seismic interpretation package
- [RINGMesh](https://github.com/ringmesh/RINGMesh) – ![C++](media/icon/cplusplus.png) Mesh manipulation of geological models
- [QGIS](http://www.qgis.com/) – GIS platform to visualize, manage, edit, analyse data, and compose printable maps

> - [MB-System](https://github.com/dwcaress/MB-System) - MB-System is an open source software package for the processing and display of bathymetry and backscatter imagery data derived from multibeam, interferometry, and sidescan sonars.

Reviewer 2: ```suggestion
- [MB-System](https://github.com/dwcaress/MB-System) – ![C](media/icon/c.png) Processing and display of bathymetry and backscatter imagery data derived from multibeam, interferometry, and sidescan sonars.
```

Reviewer 2: Please also apply these changes to conform to our style guide.
